test/metabase/api/collection_test.clj

```diff
@@ -1437,31 +1436,7 @@
                                               :descrption "My SQL Snippets"
                                               :namespace  "snippets"})))
           (finally
-            (t2/delete! Collection :name collection-name)))))
-    (testing "collection types"
```

Comment:
covered in the `enterprise/backend/test/metabase_enterprise/content_management/api/collection_test.clj`